modules/scrabble.js

```diff
@@ -107,10 +107,11 @@ module.exports = {
   },
   events: {
     message: function (bot, nick, to, text) {
-      let phrase = text.toUpperCase().split(/[^A-Z]/)
       if (text.match(bot.config.get('irc.controlChar') + 'scrabble')) return
+      let phrase = text.toUpperCase().split(/[ '"]/).filter(Boolean)
```

Comment:
What does `.filter(Boolean)` do here?

Reply:
It removes empty strings. Without it, there's a TypeError sometimes.

Reply:
I suppose a better way might be to change L115 to
```javascript
if (wordHistory.includes(word) or word=='') return
```
because there are still issues with words consisting entirely of punctuation.